### PR TITLE
Configure docker to work with storybook

### PR DESCRIPTION
**Pull request by mattwr18**

### CODE REVIEW

**Comment by alina-beck** [comment]
This is still not working for me. But maybe I'm doing it wrong? Do I have to `--build` docker to get this to work? Maybe you could provide some directions for the docker-noob-reviewers among us. ;)

File: webapp/components/Registration/CreateUserAccount.vue [hunk 1]
```diff
@@ -8,80 +8,98 @@
...
+                  primary
+                >
+                  {{ $t('actions.save') }}
+                </ds-button>
+              </template>
+            </template>
+          </ds-form>
+        </ds-flex-item>
+      </ds-flex>
+    </ds-card>
```
Comment (roschaefer) [inline webapp/components/Registration/CreateUserAccount.vue]: This is unrelated @mattwr18 please rebase.

**Comment by mattwr18** [comment]
I have rebased @roschaefer and I'm writing documentation... One issue is that it still doesn't work on localhost:3002, just on http://172.18.0.6:3002/

**Comment by mattwr18** [comment]
@alina-beck do the docs help, or that is what you were doing and it wasn't working for some reason?
to answer your question, I don't believe you do need to run `docker-compose up --build` after exposing a port in `docker-compose.yml`, but it doesn't hurt, though it takes time...

File: backend/README.md [hunk 2]
```diff
@@ -88,6 +90,40 @@ $ yarn run db:reset
...
+After you have started the application following the instructions above, in another terminal run:
+
+```bash
+$ docker-compose exec webapp yarn storybook
+```
+The output should look similar to this:
+
+![Storybook output](../.gitbook/assets/storybook-output.png)
+
+Click on the link http://172.18.0.6:3002/ to open the browser to your interactive storybook.
```
Comment (alina-beck) [inline backend/README.md]:
```suggestion
Click on the link http://localhost:3002/ to open the browser to your interactive storybook.
```

File: docker-compose.yml [hunk 3]
```diff
@@ -10,6 +10,7 @@ services:
         - "BUILD_COMMIT=${TRAVIS_COMMIT}"
     ports:
       - 3000:3000
+      - 3502:3502
```
Comment (alina-beck) [inline docker-compose.yml]:
```suggestion
      - 3002:3002
```

**Comment by mattwr18** [comment]
looks great @alina-beck... I guess we could wait to see if it works for @Tirokk 
I'll leave the decision up to you all... I think it's not a rush to get it in, but it's also not a high risk merging it in. :)
